src/web/app/components/user-notifications-list/user-notifications-list.component.html

```diff
@@ -30,6 +30,11 @@
           <div class="card-body">
             <div [innerHtml]="notificationTab.notification.message"></div>
           </div>
+          <div class="d-flex flex-row ml-3 mb-3" *ngIf="!notificationTab.isRead">
```

Comment:
Can consider changing the flex here to `flex-row-reverse` so that the button is right-aligned like the notification banner.

Reply:
Okay sure! I was following the UI mockup in the document, but I agree `flex-row-reverse` might look better!